### PR TITLE
Prepare release version 1.2.0

### PR DESCRIPTION
**Pull request by tobiashuste**
@christianhueserhzdr Could you please check this new version preparation PR. Thanks!

### CODE REVIEW

File: HISTORY.md.license [hunk 1]
```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
```
Comment (christianhueserhzdr) [inline HISTORY.md.license:2]: Dear Tobias @tobiashuste, would you like to also change the copyright year to 2022 here as well?

```suggestion
SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research (UFZ)
SPDX-FileCopyrightText: 2022 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
```

**Review by christianhueserhzdr — CHANGES_REQUESTED** [review]
Dear Tobias @tobiashuste, the PR looks fine. I just saw that you adapted the copyright year in all files but one. Shall we change that in this PR as well?

File: HISTORY.md.license (hunk 1 above)
Comment (tobiashuste) [reply HISTORY.md.license]: @christianhueserhzdr This is changed now.

**Review by christianhueserhzdr — APPROVED** [review]
Dear Tobias @tobiashuste, this PR is all fine now. Thanks. I'll approve this PR now.

**Pull request merged**